core/object/object.h

```diff
@@ -304,8 +304,10 @@ struct MethodInfo {
 
 // API used to extend in GDExtension and other C compatible compiled languages.
 class MethodBind;
+class GDExtension;
 
 struct ObjectGDExtension {
+	GDExtension *library;
```

Comment:
Should probably be initialized to `nullptr`.

Reply:
Thanks! Fixed :-)